.github/workflows/performance-test-base.yml

```diff
@@ -0,0 +1,63 @@
+name: Perftest (base branch cache)
+
+on:
+  push:
```

Comment:
@akhdrv Предлагаю переделать на   

```yml
pull_request:
    branches:
      - master
      - dev
```

Мы сейчас во всех так своих workflows делаем.
    
> If a workflow run is triggered for a pull request, it can also restore caches created in the base branch, including base branches of forked repositories.

Reply:
нет, этот воркфлоу нужно гонять на пуш в базовую ветку. это не надо на каждый PR прогонять